vignettes/templates.Rmd

```diff
@@ -0,0 +1,350 @@
+---
+title: "Creating Template tfrmts"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating Template tfrmts}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(tfrmt)
+library(dplyr)
+library(gt)
+library(tidyr)
+```
+
+A core design element of `tfrmt` is to offer the ability to layer tfrmts together. This ability provides an opportunity for organizations to build template table formats that can be shared, improved, and reused across multiple projects. Through layering the users can customize the templates to create a specific table. 
+
+The creation of a template is based on the standard development of a `tfrmt`, but wrapping its creation into a function. using the `layer_tfrmt` function. We will go through their applications below.  
+
+## Creating A Standard tfrmt
+
+A template is based around creating a standard `tfrmt`, and then writing a function around it to provide the ability to layer multiple templates. 
+
+We will create an Adverse Events table template in this example. This requires there to be some standards that can be expected that allows the construction of the basic `tfrmt`:
```

Comment:
It might be better to move the  first sentence "we will create an AE table template.." to before the two sections to set the example up for the reader

Reply:
```suggestion
The first step to creating an Adverse Events table template is to determine the constants in the format. This requires there to be some standards that can be expected that allows the construction of the basic `tfrmt`:
```